Resources/Prototypes/Entities/Clothing/Belt/base_clothingbelt.yml

```diff
@@ -23,6 +23,13 @@
     capacity: 40
     equipSound:
       path: /Audio/Items/belt_equip.ogg
+  - type: MeleeWeapon
+    attackRate: 1.1
+    range: 1.65
+    damage:
+      types:
+        Blunt: 5
+        Slash: 2
```

Comment:
Why was the Blunt and Slash added, the intent was that it should be 1 blunt 1 burn for the damage?

Reply:
ah yeah, its just that  was having issues with it error wise let me commit for it to be 1/1.

Reply:
There shouldn't be any errors doing from doing that; you are probably missing the actual problem

Reply:
the actual error was that i had forgotten to add the damage types on the original commit, those 5 blunt and 2 slash was to test if it fixed it.